frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -604,20 +602,20 @@ common_settings.document_metadata_location = {
         genMetadataFolderMenuItem("dir"),
         genMetadataFolderMenuItem("hash"),
         { -- hash-based metadata count / TextViewer
-            text = (function()  -- seems like this is running whenever the root menu is opened, not just sdr storage?
-                local hash_file_infos = DocSettings.getHashDirSdrInfos()
-                local text = T(_("Show %1 documents with hash-based metadata"), #hash_file_infos)
-                if #hash_file_infos == 0 then
-                    return T(_("No documents with hash-based metadata"), #hash_file_infos)
-                end
-                if G_reader_settings:readSetting("document_metadata_folder") ~= "hash" then
-                    return "⚠ "..text  -- I'm not sure how to instantly update this after changing to another storage method radio bullet
+            text_func = function()
+                local hash_text = _("Find 1+ documents with hash-based metadata")
```

Comment:
May be just (as before): `Show documents with hash-based metadata` ?

Reply:
@poire-z So do you think [this](https://github.com/koreader/koreader/pull/10945#issuecomment-1743492589) is a nonissue? If it's clear how you suggested I'm fine to update it.

Reply:
Dunno, I just find "Find 1+" strange - compared to our usual wording of things :)
"Show documents..." (if 1+) and "No documents..." (if 0) might be enough ? Moreover if there's the ⚠ when it really matters (storage != "hash"). Our menu invites to be tapped :)

Reply:
Okay, sounds good.